lib/govuk_content_schemas/frontend_schema_generator.rb

```diff
@@ -12,6 +12,7 @@ class GovukContentSchemas::FrontendSchemaGenerator
     redirects
     routes
     update_type
+    change_note
```

Comment:
Sorry, totally inverted the logic of this in my head. This is a blacklist not a whitelist. 👍